src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,38 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DatabaseManagerTest {
+
+    private DatabaseManager manager;
+
+    @Before
+    public void initialize() {
+        manager = new DatabaseManager();
+    }
+
+
+    @Test
+    public void connectionWithValidParametersHasToReturnTrue() {
+        assertTrue(manager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+    }
+
+    @Test
+    public void connectionWithWrongDatabaseNameHasToReturnFalse() {
```

Comment:
Common word is 'should'. .shouldReturnTrue()..

Reply:
I asked my English teacher about should and has to.